src/lib/mdns/ServiceNaming.h

```diff
@@ -27,7 +27,7 @@
 
 namespace chip {
 namespace Mdns {
-constexpr size_t kMaxSubtypeDescSize           = 16; // max 16 char service name
+constexpr size_t kMaxSubtypeDescSize           = 19; // _I service subtype.
```

Comment:
Suggest giving the worst case example that justifies the 19

Reply:
added in next commit.